CMakeLists.txt

```diff
@@ -2738,6 +2746,9 @@ elseif(UNIX AND NOT APPLE)
     Qt${QT_VERSION_MAJOR}::DBus
   )
 elseif(WIN32)
+    target_link_libraries(mixxx-lib PRIVATE
+    Comctl32
```

Comment:
this is in the Qt related block. 
No problem, but we have also 
`target_link_libraries(mixxx-lib PRIVATE shell32)` and than `shell32` later in the static build block. 
Can you join that?